ocaml/xapi/extauth_plugin_ADwinbind.ml

```diff
@@ -245,15 +541,42 @@ module AuthADWinbind : Auth_signature.AUTH_MODULE = struct
   let authenticate_ticket tgt =
     failwith "extauth_plugin authenticate_ticket not implemented"
 
-  (* subject_id get_subject_identifier(string subject_name)
-
-      Takes a subject_name (as may be entered into the XenCenter UI when defining subjects --
-      see Access Control wiki page); and resolves it to a subject_id against the external
-      auth/directory service.
-      Raises Not_found (*Subject_cannot_be_resolved*) if authentication is not succesful.
-  *)
-  let get_subject_identifier _subject_name =
-    "get_subject_identifier To be implemented in CP-36087"
+  let query_subject_information_group (name : string) (gid : int) (sid : string)
+      =
+    [
+      ("subject-name", name)
+    ; ("subject-gid", string_of_int gid)
+    ; ("subject-sid", sid)
+    ; ("subject-is-group", string_of_bool true)
```

Comment:
why not just `; ("subject-is-group", "true")`

Reply:
they are functionally equivalent